lib/commands/start.js

```diff
@@ -45,6 +46,13 @@ module.exports = BaseCommand.extend({
         // TODO: rethink this
         return this.runCommand('config', 'paths.contentPath', path.join(process.cwd(), 'content'), {environment: environment})
             .then(function () {
+                knexMigrator = new KnexMigrator();
```

Comment:
see below comment - I see why you didn't instantiate knex-migrator until later on in the code, but at the same time moving the variables down makes more perf sense imo